lib/matplotlib/__init__.py

```diff
@@ -1476,7 +1476,6 @@ def _jupyter_nbextension_paths():
 default_test_modules = [
     'matplotlib.tests.test_png',
     'matplotlib.tests.test_units',
-    'matplotlib.tests.test_widgets',
```

Comment:
When switching to the automatic fixture and dropping any `@cleanup`s, the tests fail with nose because they don't get that setup done. However, see next comment.